test/user-profile/helpers.unit.spec.js

```diff
@@ -0,0 +1,85 @@
+import { expect } from 'chai';
+import _ from 'lodash';
+
+import {
+  formTitles,
+  formLinks,
+  handleIncompleteInformation,
+  handleNonSIPEnabledForm,
+  sipEnabledForms,
+} from '../../src/js/user-profile/helpers';
+
+import fullSchema1010ez from '../../src/js/hca/config/form';
+import fullSchema1990 from '../../src/js/edu-benefits/1990-rjsf/config/form';
+import fullSchema1990e from '../../src/js/edu-benefits/1990e/config/form';
+import fullSchema1990n from '../../src/js/edu-benefits/1990n/config/form';
+import fullSchema1995 from '../../src/js/edu-benefits/1995/config/form';
+import fullSchema5490 from '../../src/js/edu-benefits/5490/config/form';
+import fullSchema5495 from '../../src/js/edu-benefits/5495/config/form';
+import fullSchema527EZ from '../../src/js/pensions/config/form';
+import fullSchema530 from '../../src/js/burials/config/form';
+
+import schemas from 'vets-json-schema/dist/schemas';
+
+// Maps schema id to config id
+const schemaToConfigIds = {
+  '10-10EZ': '1010ez',
+  '21P-527EZ': '21P-527EZ',
+  '21P-530': '21P-530',
+  '22-1990': '1990',
+  '22-1990E': '1990e',
+  '22-1990N': '1990n',
+  '22-1995': '1995',
+  '22-5490': '5490',
+  '22-5495': '5495',
+  '40-10007': 'N/A', // the config for this form does not contain a formId property
+  definitions: 'N/A'
+};
+
+describe('profile helpers:', () => {
+  describe('formTitles', () => {
+    it('should have title information for each verified form', () => {
+      sipEnabledForms.forEach(form => {
+        expect(formTitles[form]).to.exist;
+      });
+    });
+  });
+  describe('formLinks', () => {
+    it('should have link information for each verified form', () => {
+      sipEnabledForms.forEach(form => {
+        expect(formLinks[form]).to.exist;
+      });
+    });
+  });
+  describe('sipEnabledForms', () => {
+    it('should include all and only SIP enabled forms', () => {
+      const configs = [
+        fullSchema1010ez,
+        fullSchema1990,
+        fullSchema1990e,
+        fullSchema1990n,
+        fullSchema1995,
+        fullSchema5490,
+        fullSchema5495,
+        fullSchema527EZ,
+        fullSchema530
+      ];
+      const allFormIds = Object.keys(schemas);
+      const allMappedIds = Object.keys(schemaToConfigIds);
+      const sipEnabledConfigs = configs.filter(config => !config.disableSave);
+      const sipEnabledFormIds = sipEnabledConfigs.map(sipEnabledConfig => sipEnabledConfig.formId);
+      expect(_.isEqual(allFormIds, allMappedIds)).to.be.true;
```

Comment:
You should be able to use something like `expect(allFormIds).to.equal(allMappedIds)`